plugins/rikki/loungeviews/components/playoffoverview/default.htm

```diff
@@ -148,7 +147,7 @@ <h6>Team Description:</h6>
                                 <div class="state">
                                     <time datetime="{{match['model'].wbp | date('m/d/Y', __SELF__.timezone)}}" dateformat="custom_medium" timeformat="none">{{match['model'].wbp | date('d M', __SELF__.timezone)}}</time>
                                     <div class="time"><time datetime="{{match['model'].wbp | date('m/d/Y', __SELF__.timezone)}}" dateformat="none" timeformat="short">{{match['model'].wbp | date('H:i', __SELF__.timezone)}}</time></div>
-                                    <div>Timezone: {{__SELF__.timezone}} {{__SELF__.timezoneOffset}}</div>
+                                    <div>({{__SELF__.timezoneOffset}})</div>
```

Comment:
This is the one actual change in here because when looking at unscheduled playoff games the timezone is huge

Reply:
I think for this page it would be even better to just show the timezone in a single place at the top or bottom instead of repeating the same information for every scheduled match.

Reply:
Excellent point.  I put it here because time and date might not actually be displayed anywhere on the page but I think your suggestion makes the page much, much cleaner and a timezone reminder without times will still be fine.

This has been updated now.